lxmls/classifiers/svm.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, nr_epochs=10, initial_step=1.0, alpha=1.0, regularizer=1.0, s
         np.random.seed(seed)
 
     def train(self, x, y):
-        self.params_per_round = []
+        #self.params_per_round = []
```

Comment:
@Askinkaty can you elaborate about this decision?. With this change the parameters will be accumulated over various training instances.

If you think this should stay please delete the line entirely.

Reply:
@ramon-astudillo agree, didn't think about accumulating